packages/obojobo-document-engine/ObojoboDraft/Chunks/IFrame/viewer-component.js

```diff
@@ -190,16 +193,24 @@ export default class IFrame extends React.Component {
 								{ms.src === null ? null : <Button>View Content</Button>}
 							</div>
 						)}
-						<Controls
-							newWindowSrc={src}
-							controlsOptions={controlsOpts}
-							isZoomAbleToBeReset={zoomValues.isZoomDifferentFromInitial}
-							isUnableToZoomOut={isAtMinScale}
-							reload={this.boundOnReload}
-							zoomIn={this.onClickSetZoom.bind(this, parseFloat((zoom + 0.1).toFixed(2)))}
-							zoomOut={this.onClickSetZoom.bind(this, parseFloat((zoom - 0.1).toFixed(2)))}
-							zoomReset={this.boundOnZoomReset}
-						/>
+						{!isShowing ? null : (
+							<Controls
+								newWindowSrc={src}
+								controlsOptions={controlsOpts}
+								isZoomAbleToBeReset={!zoomValues.isZoomAtDefault}
```

Comment:
Maybe change this variable to canResetZoom? isZoomAbleToBeReset is a little long and hard to understand

Reply:
Agree. A by-product of trying to enforce the `is...` naming scheme for boolean variables. Perhaps those should be relaxed?

Reply:
I'm thinking of `isZoomResettable`?